nurseconnect/templates/core/tags/section_listing_menu.html

```diff
@@ -35,6 +37,10 @@ <h1 class="HeroTout-headline">
           {% trans "Log out" %}
         </a>
       </li>
+      {% else %}
+      <li class="Menu-navItem" role="presentation">
+        <a href="{% url "auth_login" %}?next={{request.path}}" class="Menu-navLink" role="menuitem">{% trans "Login" %}</a>
+      </li>
+      {% endif %}
     </ul>
```

Comment:
shouldn't the ul and nav be inside the if

Reply:
@codiebeulaine  no that ul is declared before the if